### DISCUSSION TITLE
Add new features for power schemas and support EnableUlps

### DISCUSSION THREAD

**Discussion by RaZer0k** [seerge/g-helper#705] [Ideas]
Add a new option inside Extra to enable or disable "EnableUlps" like Afterbuner
Add a option inside each Modes to select a power schema... so you dont need to add it editing config.json

**Comment by seerge** [comment]
@RaZer0k I don't know if this possible even to control 'ULPS" via nvidia api.

As for power schemas, I think it will make UI more complicated and raise more questions. App has default behavior of adjusting power modes with each "profile". Power plans are almost obsolete in windows 11, and cause only confusion :)

**Reply by Provinomico** [reply]
In relation to power schemas/profiles, would it be possible to add the custom ones that are created on the 'Fans+Power' window, to the context panel that opens when you right click the tray icon?

![image](https://github.com/seerge/g-helper/assets/137456477/0a32bb90-b678-4b4f-8889-dcf85a9f63ba)

**Reply by seerge** [reply]
It's a bit difficult actually ,as  currently each context-menu item is literally linked to a corresponding button in main window. And it's also sort of consistent with rest of UI. 

Do you know that you can also customize all 3 default modes as well? Idea of "Custom" modes, was mainly for people who have a lot of specific use-cases and need more than 3 modes in total.
